flow/project.py

```diff
@@ -715,32 +846,42 @@ def _submit_cmd(self, entrypoint, ignore_conditions, parallel, job=None):
         else:
             return cmd.strip()
 
-    def _run_cmd(self, entrypoint, operation_name, operation, directives, job):
+    def _run_cmd(self, entrypoint, operation_name, operation, directives, jobs):
+        jobstr = deepcopy(jobs)
+        if not isinstance(jobs, signac.contrib.job.Job):
+            if not isinstance(jobs, list):
```

Comment:
When would we have a collection that is neither a list or a job?

Reply:
I agree with the second comment. The functionality here is same as the idea we all developed.

Reply:
Thank you for pointing this out, I'll edit the code.